### ISSUE TITLE
Could this be used as a satellite instead of a Raspberry Pi Zero w v2

### ISSUE THREAD

**Issue by ionultd** [mkvenkit/mico#1]
Could this be used as a satellite instead of a Raspberry Pi Zero w v2

**Comment by mahesh-electronut** [comment]
> Could this be used as a satellite instead of a Raspberry Pi Zero w v2

Sorry, I don't understand what you mean by satellite.

**Comment by ionultd** [comment]
Oh right lol. I mean could this be used to run Picovoice wake word and then send instructions to voice AI.

**Comment by mahesh-electronut** [comment]
Yes, that's a possibility, since there's TensorFlow Lite support for RP2040 now.

**Comment by wildblue** [comment]
You wrote that you are producing them right now. Where can they be ordered and for how much euros/dollars?

**Comment by mahesh-electronut** [comment]
> You wrote that you are producing them right now. Where can they be ordered and for how much euros/dollars?

I am preparing for crowdfunding production of Mico. Please follow me on twitter for updates: https://twitter.com/mkvenkit

**Comment by krasi-georgiev** [comment]
I am also interested .
messaged you on tweeter for some other things related to the project.

**Comment by sixtyfive** [comment]
If there was a version that had 2 microphones a few centimetres apart from one another that would make for an even better voice satellite. Even on a Raspi, given that there's also [PiDTLN](https://github.com/SaneBow/PiDTLN) . Re Picovoice, take a look at [micro-wake-word](https://github.com/OHF-Voice/micro-wake-word) as well.
